src/MoBi.Presentation/Tasks/SimulationRunner.cs

```diff
@@ -45,15 +51,22 @@ public class SimulationRunner : ISimulationRunner
          _displayUnitUpdater = displayUnitUpdater;
          _simModelManagerFactory = simModelManagerFactory;
          _keyPathMapper = keyPathMapper;
+         _entityValidationTask = entityValidationTask;
       }
 
       public void RunSimulation(IMoBiSimulation simulation, bool defineSettings = false)
       {
          addPersitableParametersToOutputSelection(simulation);
+         if (!_entityValidationTask.Validate(simulation))
```

Comment:
@rwmcintosh This is really the only change besides dll update. Plugging in the validation task to ensure that the simulation is valid before running it. This should have been done already but this was missing